pandas_transform/notebooks/dataframe_example.ipynb

```diff
@@ -0,0 +1,2363 @@
+       "      <td>covid</td>\n",
+       "      <td>emergency dept</td>\n",
+       "      <td>positive</td>\n",
+       "      <td>patient</td>\n",
+       "      <td>0.1</td>\n",
+       "      <td>0</td>\n",
+       "      <td>23.97</td>\n",
+       "      <td>1</td>\n",
+       "      <td>commercial</td>\n",
+       "      <td>inpatient</td>\n",
+       "      <td>0.3</td>\n",
+       "      <td>5.5</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         mrn first_name   last_name  gender  pan_day test_id  \\\n",
+       "27   5002114      azzak       tully    male       10   covid   \n",
+       "91   5011391        zei       umber  female       11   covid   \n",
+       "100  5000902       owen    seaworth    male       12   covid   \n",
+       "440  5005030     allard      rivers    male       18   covid   \n",
+       "472  5007953       anya  westerling  female       18   covid   \n",
+       "\n",
+       "          clinic_name    result  demo_group  age  drive_thru_ind  ct_value  \\\n",
+       "27   inpatient ward b  positive     patient  0.9               0     39.55   \n",
+       "91       s  care ntwk  positive     patient  0.1               0     39.83   \n",
+       "100    emergency dept  positive     patient  0.1               0     32.96   \n",
+       "440      clinical lab  positive  misc adult  0.0               1     16.30   \n",
+       "472    emergency dept  positive     patient  0.1               0     23.97   \n",
+       "\n",
+       "     orderset payor_group   patient_class  col_rec_tat  rec_ver_tat  \n",
+       "27          0  government     observation          0.6          2.7  \n",
+       "91          0  government      outpatient          5.2          4.8  \n",
+       "100         1  government       emergency          0.3          4.5  \n",
+       "440         0  unassigned  not applicable          2.4          5.6  \n",
+       "472         1  commercial       inpatient          0.3          5.5  "
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "is_positive_infant = (covid_testing.loc[:,\"age\"]<1) & (covid_testing.loc[:,\"result\"] == \"positive\")\n",
+    "\n",
+    "infant_positive = covid_testing.loc[is_positive_infant,:]\n",
+    "infant_positive.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a9a3750-5bb7-422d-a23d-a85d12aeb6af",
+   "metadata": {},
+   "source": [
+    "## Creating new columns\n",
+    "\n",
+    "Sometimes the data in a DataFrame isn't in the best form for us to use. Maybe it is a string that is too long or unclear, or maybe the units aren't the unit we will ultimately want to use to analize the data. If we only cared about children under age 1, like in the `infant_positive` DataFrame, it might make sense to have age in units of months instead of years.\n",
```

Comment:
In this line, it should be "...or maybe the units aren't the **units** that we will ultimately..."